packages/components/src/components/radio-button/radio-button.tsx

```diff
@@ -81,21 +81,25 @@ export class RadioButton {
   }
 
   handleCheckedChange = (event: any) => {
-    this.checked = event.target.checked;
-    // I don't think this is ever going to be `false` but well...
-    if (this.checked) {
-      this.uncheckSiblings();
+    if (!this.disabled) {
```

Comment:
i think this one is not really needed but i guess it doesn't really hurt either 🤓

Reply:
I checked the button gets clickable if we do not add this.

Reply:
oh ok, i thought ` cursor: not-allowed;` would already prevent that on the input field